kernel/aster-nix/src/process/signal/sig_mask.rs

```diff
@@ -1,61 +1,115 @@
 // SPDX-License-Identifier: MPL-2.0
 
+//! Signal sets and atomic masks.
+//!
+//! A signal set is a bit-set of signals. A signal mask is a set of signals
+//! that are blocked from delivery to a thread. An atomic signal mask
+//! implementation is provided for shared access to signal masks.
+
+use core::{
+    fmt::LowerHex,
+    ops,
+    sync::atomic::{AtomicU64, Ordering},
+};
+
 use super::{constants::MIN_STD_SIG_NUM, sig_num::SigNum};
 use crate::prelude::*;
 
-/// A signal mask.
-pub type SigMask = SigSet;
```

Comment:
I suggest keeping `SigMask` as it conveys the intention more clearly, which is the one good use of type alias. People may confuse as to whether the signals within a `SigSet` are interesting (unmasked) or uninteresting (masked).

For example, this is the original definition of `SigEventsFilter`.

```rust
impl SigEventsFilter {
    pub fn new(mask: SigMask) -> Self {
```

From the type of the constructor argument, it is clear than the filter will filter out the mask, only allowing the unmask signals. But if it is changed to the following form proposed by this PR, then the intention will be less clear: does the filter want the given `set` or not?

```rust
impl SigEventsFilter {
    pub fn new(set: SigSet) -> Self {
```

You may argue that naming the variable `mask` would resolve the ambiguity. Yes, this helps. But having both the type and variable names can do this job better. And sometimes we cannot use the help of variable names when the type name is used alone.

```rust
pub struct SigEventsFilter(SigSet);
```